Mathlib/Topology/MetricSpace/Basic.lean

```diff
@@ -190,3 +190,101 @@ instance SeparationQuotient.instMetricSpace {α : Type u} [PseudoMetricSpace α]
     surjective_mk.forall₂.2 dist_edist
 
 end EqRel
+
+namespace PseudoEMetricSpace
+
+open ENNReal
+
+variable {X : Type*} (m : PseudoEMetricSpace X) (d : X → X → ℝ≥0∞) (hd : d = edist)
+
+/-- Build new pseudoemetric space from an old one where the edistance uniform structure is provably
+(but typically non-definitionally) equal to some given distance structure. Below are convenience
+versions for PseudoMetric, Emetric and Metric spaces. -/
```

Comment:
The docstring of a theorem will typically appear when hovering above a use of the theorem. So, references to "below" or "above" don't really make sense in this context, and the docstrings should be self-contained. So you could say in this one "We also provide convenience versions for ..." (without the Below), and the other ones should essentially be copies of this one, with the same amount of detail.